scenarios/asset-store-migration/README.md

```diff
@@ -0,0 +1,77 @@
+# Scenario: Asset store migration
+This scenarios shows how to register a machine and notarize assets to verify the functionality of the store migration from planetmint-god `v0.9.7` to `v0.10.0`.
+
+## Prerequisites
+Download planetmint-god v0.9.7 binary
+
+`wget https://github.com/planetmint/planetmint-go/releases/download/v0.9.7/planetmint-god`
+
+Download planetmint-god v0.10.0 binary
+
+`wget https://github.com/planetmint/planetmint-go/releases/download/v0.10.0/planetmint-god`
+
+**Caveat:** Give the binaries distinct names.
+
```

Comment:
I needed to make the binaries executable after downloading.

Please add the line `chmod 755 planetmint-god planetmint-god-v0100`